### PR TITLE
LED-71 Transaction Import

### PR DESCRIPTION
**Pull request by cjm721**
Basic functionality is there but need to go through and do CSS again.

Notable changes:
* Each dropdown is it's own class (Account/Converter).
* FileSelectorButton used instead of having that code in multiple places.

Made a few changes to DbController so that it handles Nulls correctly.

### CODE REVIEW

File: src/main/java/ledger/controller/ImportController.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+package ledger.controller;
+
+import ledger.controller.register.TaskWithArgs;
+import ledger.database.IDatabase;
+import ledger.database.entity.Account;
+import ledger.database.entity.Transaction;
+import ledger.io.importer.Importer;
+import ledger.io.input.*;
```
Comment (shellajt) [inline src/main/java/ledger/controller/ImportController.java]: Can we get more specific than input.*?

File: src/main/java/ledger/controller/ImportController.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+import java.io.File;
+import java.util.List;
+
+/**
+ * Created by CJ on 10/23/2016.
```
Comment (shellajt) [inline src/main/java/ledger/controller/ImportController.java]: Class documentation

File: src/main/java/ledger/user_interface/ui_controllers/ConverterDropdown.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package ledger.user_interface.ui_controllers;
+
+import javafx.collections.FXCollections;
+import javafx.scene.control.ChoiceBox;
+import ledger.controller.ImportController;
+import ledger.database.entity.Transaction;
+import ledger.io.input.IInAdapter;
+
+/**
+ * Created by CJ on 10/25/2016.
```
Comment (shellajt) [inline src/main/java/ledger/user_interface/ui_controllers/ConverterDropdown.java]: Class Documentation

File: src/main/java/ledger/controller/ImportController.java (hunk 1 above)
Comment (cjm721) [inline src/main/java/ledger/controller/ImportController.java]: It is auto formatting to that since everything in input.* is being used.

**Pull request merged**